ControlFlow/Program.cs

```diff
@@ -48,39 +73,88 @@ public static string LandscapeOrPortrait(int width, int height)
         // Suspended" instead. Don't Worry about input validation.
         public static string SpeedTrap(int speedLimit, int speedCar)
         {
-            return default;
+             int demeritPoints = (speedCar - speedLimit) / 5;
+
+            if (speedCar > speedLimit && demeritPoints <= 12)
+            {
+                return demeritPoints.ToString();
+            }
+            else if (demeritPoints > 12)
+            {
+                return "License Suspended";
+            }
+            else
+            {
+                return "Okay";
+            }
```

Comment:
Really liked the logic.

I recommend like above, to use fewer return statements.